cloud: add release notes for 20240903 (#18774)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

- Support exporting data from TiDB Serverless clusters using the [TiDB Cloud console](https://tidbcloud.com/). 
  
    Previously, TiDB Cloud only supported exporting data using the [TiDB Cloud CLI](/tidb-cloud/cli-reference.md). Now, you can easily export data from TiDB Serverless clusters to local files and Amazon S3 in the [TiDB Cloud console](https://tidbcloud.com/). 
  
    For more information, see [Export Data from TiDB Serverless](/tidb-cloud/serverless-export.md) and [Configure External Storage Access for TiDB Serverless](/tidb-cloud/serverless-external-storage.md).

- Enhance the connection experience for [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters.

    - Revise the **Connect** dialog interface to provide TiDB Dedicated users with a more streamlined and efficient connection experience.
    - Introduce a new cluster-level **Networking** page to simplify network configuration for your cluster.
    - Replace the **Security Settings** page with a new **Password Settings** page and move IP access list settings to the new **Networking** page.
  
  For more information, see [Connect to TiDB Dedicated](/tidb-cloud/connect-to-tidb-cluster.md).

> 

Reviewer: ```suggestion

- Refine the **Import** page with a clearer layout, unified steps for [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless) and [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters, providing more user-friendly and efficient experience.

    For more information, see [Import data from files to TiDB Cloud](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud).

```

Reviewer 2: LGTM

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.

## September 3, 2024

> 

Reviewer: ```suggestion
```

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.

## September 3, 2024

**General changes**

> - Enhance the connection experience for TiDB Dedicated

Reviewer 3: ```suggestion
- Enhance the connection experience for [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters.
```

    - Revise the **Connect** dialog interface to provide TiDB Dedicated users with a more streamlined and efficient connection experience.

>     - Introduce a new **Networking** page at the cluster level in TiDB Dedicated to simplify network configuration for your cluster.

Reviewer: ```suggestion
    - Introduce a new cluster-level **Networking** page in TiDB Dedicated to simplify network configuration for your cluster.
```

>     - Refine the Connect dialog interface to offer TiDB Dedicated users a smoother and more efficient connection experience.
>     - TiDB Dedicated introduces "Networking page" in Cluster level to simplify the network configuration for your cluster.
>     - Remove the previous "Security Settings" page, and add a new "Password Setting" page. The IP access list settings move from "Security Settings" page to new "Networking" page.

Author: ```suggestion
    - Refine the **Connect** dialog interface to offer TiDB Dedicated users a smoother and more efficient connection experience.
    - TiDB Dedicated introduces the **Networking** page in cluster level to simplify the network configuration for your cluster.
    - Remove the **Security Settings** page, and add a new "Password Setting" page. Move the IP access list settings from the **Security Settings** page to a new **Networking** page.
```

> - Revise the **Connect** dialog interface to provide TiDB Dedicated users with a more streamlined and efficient connection experience.
> - Introduce a new **Networking** page at the cluster level in TiDB Dedicated to simplify network configuration for your cluster.
> - Replace the **Security Settings** page with a new **Password Setting** page. The IP access list settings have been moved from the **Security Settings** page to the new **Networking** page.

Author: ```suggestion
    - Revise the **Connect** dialog interface to provide TiDB Dedicated users with a more streamlined and efficient connection experience.
    - Introduce a new **Networking** page at the cluster level in TiDB Dedicated to simplify network configuration for your cluster.
    - Replace the **Security Settings** page with a new **Password Setting** page. The IP access list settings have been moved from the **Security Settings** page to the new **Networking** page.
```

>     - Replace the **Security Settings** page with a new **Password Setting** page. The IP access list settings have been moved from the **Security Settings** page to the new **Networking** page.

Reviewer: ```suggestion
    - Replace the **Security Settings** page with a new **Password Settings** page. The IP access list settings have been moved from the **Security Settings** page to the new **Networking** page.
```

> **General changes**
> 
> - Enhance the connection experience for [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters.
> 
>     - Revise the **Connect** dialog interface to provide TiDB Dedicated users with a more streamlined and efficient connection experience.
>     - Introduce a new cluster-level **Networking** page to simplify network configuration for your cluster.
>     - Replace the **Security Settings** page with a new **Password Settings** page and move IP access list settings to the new **Networking** page.
>   
>   For more information, see [Connect to TiDB Dedicated](/tidb-cloud/connect-to-tidb-cluster.md).

Reviewer: ```suggestion
```

**Console changes**

> - Support exporting data from TiDB Serverless clusters in the TiDB Cloud console. 

Reviewer 3: ```suggestion
- Support exporting data from TiDB Serverless clusters using the [TiDB Cloud console](https://tidbcloud.com/). 
```

>     Previously, TiDB Cloud only supported exporting data using the [TiDB Cloud CLI](/tidb-cloud/cli-reference.md). Now, you can export data from TiDB Serverless clusters to local files and Amazon S3 in the [TiDB Cloud console](https://tidbcloud.com/). 

Author: ```suggestion
    Previously, TiDB Cloud only supported exporting data using the [TiDB Cloud CLI](/tidb-cloud/cli-reference.md). Now, you can easily export data from TiDB Serverless clusters to local files and Amazon S3 in the [TiDB Cloud console](https://tidbcloud.com/). 
```

  
    For more information, see [Export Data from TiDB Serverless](/tidb-cloud/serverless-export.md) and [Configure External Storage Access for TiDB Serverless](/tidb-cloud/serverless-external-storage.md).

> 

Reviewer: ```suggestion

- Enhance the connection experience for [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters.

    - Revise the **Connect** dialog interface to provide TiDB Dedicated users with a more streamlined and efficient connection experience.
    - Introduce a new cluster-level **Networking** page to simplify network configuration for your cluster.
    - Replace the **Security Settings** page with a new **Password Settings** page and move IP access list settings to the new **Networking** page.
  
  For more information, see [Connect to TiDB Dedicated](/tidb-cloud/connect-to-tidb-cluster.md).

```
